### ISSUE TITLE
Order on out_of_stock product, with an order_state that is logable = 0, add physical_quantity in stock_available

### ISSUE THREAD

**Issue by groukx** [PrestaShop/PrestaShop#28420]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

A product that has stock 0 (product out_of_stock = 1)
![Capture d’écran (27)](https://user-images.githubusercontent.com/13377687/166918758-b88b252f-5c5a-4722-8e70-3a0b5ada31a6.png)
An order is placed on this product with wiretransfer (order_state with logable = 0)
![Capture d’écran (28)](https://user-images.githubusercontent.com/13377687/166918809-57a6df83-2259-47a1-ac8a-bd9b055403ba.png)
A physical quantity is added from nowhere and also without a movement so the integrity of the database is compormised

### Expected behavior

physical_qty | reserved_qty | Available_qty
0 | 1 | -1

### Steps to reproduce

order on a product with out_of_stock setting set to 1 and that has 0 quantity in stock
The order_state has it's logable value set to 0

### PrestaShop version(s) where the bug happened

all 1.7

### PHP version(s) where the bug happened

7.4

**Comment by groukx** [comment]
My supplier_orders are filled automatically, but as the product become available on stock, it never trigger the supply.
Let me know if i can help as a web developper

**Comment by HanaRebaiQA** [comment]
Hello @groukx 

Thanks for your report.
Could you please describe more the steps to reproduce in details with a screen-shot / screen-record to can reproduce the issue clearly ?

By the way, Is your issue similar to this one [#24531](https://github.com/PrestaShop/PrestaShop/issues/24531) ?

Thank you!

**Comment by groukx** [comment]
Hello @HanaRebaiQA ,
Thank you for the delay of the reply. No, nothing related to #24531
The bug that i report add quantity of product in stock from nowhere without movement.

It is very easy to reproduce.
Enable the ability to order a product out of stock, Have this product out of stock. Order it with an order_state that has it's logable column set to 0 like wiretransfer

**Comment by groukx** [comment]
My temporary workarround is to have setted up wiretransfer has the PS_OS_ERROR in configuration  which is ignored from reservation. I don't have downside effect except, I can't use others pending payment status. Those status were not reserved in prestashop 1.6

**Comment by hibatallahAouadni** [comment]
Hello @groukx 

Thanks for your patience :pray: 
Could you please provide more details and clearer steps to reproduce :pray: 
> order on a product with out_of_stock setting set to 1 and that has 0 quantity in stock
The order_state has it's logable value set to 0

What do you mean by `logable value set to 0`??

Thanks!

**Comment by groukx** [comment]
Hi @hibatallahAouadni 
in order_state table there is a column logable.
All the pending payments status has this column set to 0 as prestashop default value

I am currently installing the dev branch to see if it is still the case in PrestaShop8 version

**Comment by hibatallahAouadni** [comment]
Hello @groukx 

> in order_state table there is a column logable.
All the pending payments status has this column set to 0 as prestashop default value

I see what you mean and this is the expected behavior:

![image](https://user-images.githubusercontent.com/23308427/167469872-47a7fca8-a777-476e-9e5d-910af78751cc.png)

Now, could you please provide clear and concise steps to reproduce the issue :pray: 
Please I need these steps to help you fix the issue ASAP :pray: 
Otherwise, just share a screen record :relaxed: 
Waiting for your feedback.

Thanks!

**Comment by groukx** [comment]
Hi @hibatallahAouadni ,
I don't want to show public capture of my production website.
I just tested it with prestashop 8 and the bug does not occur. 
I go to install a fresh 1.7.8. Stable realease and come back to you

**Comment by hibatallahAouadni** [comment]
@groukx Glad to hear 😄 
Waiting for your feedback!

**Comment by groukx** [comment]
Sorry @hibatallahAouadni,
The bug does not occur in prestashop 1.7.8. I found out that it is related to a module that we are using. Now I have prestashop 8, prestashop 1.7.8.5 installed locally, I would be more precise to report another bug that I have related to stock movements when product added to an order from BO that occur in the fresh 1.7.8.5 install and that is fixed in prestashop 8.
I close this issue, Thnak you for your help
